doc/machine-auto.rst

```diff
@@ -0,0 +1,33 @@
+machine_dict: 
+    | type: ``dict``
+    | argument path: ``machine_dict``
+
+    batch_type: 
+        | type: ``str``
+        | argument path: ``machine_dict/batch_type``
+
+        The batch job system type. Option: Slurm, PBS, LSF, Shell, DpCloudServer
```

Comment:
current dpgen (also deepks) implementation do not need those choices to be capitalized. I understand this relates to the implementation of dpdispatcher but we'd better follow the old convention and make it case insensitive.